### PR TITLE
Set PKCS11_ENGINE_PATH when using aktualizr with HSM

### PR DESCRIPTION
**Pull request by lbonn**
Depends on https://github.com/advancedtelematic/aktualizr/pull/1471.

### CODE REVIEW

**Comment by patrickvacek** [comment]
This looks fine, but is there a way to make it configurable if a user wants to use a different HSM or pkcs11 provider?

**Comment by lbonn** [comment]
@patrickvacek I think it's more about system configuration than user choice as it should be fixed when using libp11 (which aktualizr assumes).

That said, it would probably need to be changed when using older versions of openssl so I'll add it as a variable in the recipe so it can be easily overriden.

File: recipes-sota/aktualizr/aktualizr-device-prov-hsm.bb [hunk 1]
```diff
@@ -14,7 +14,7 @@ DEPENDS = "aktualizr"
 # If the config file from aktualizr used here is changed, you will need to bump
 # the version here because of SIGGEN_EXCLUDE_SAFE_RECIPE_DEPS!
 PV = "1.0"
-PR = "6"
```
Comment (lbonn) [inline recipes-sota/aktualizr/aktualizr-device-prov-hsm.bb]: Got bitten by this...

Comment (patrickvacek) [reply recipes-sota/aktualizr/aktualizr-device-prov-hsm.bb]: Argh, sorry. That is annoying.

**Pull request merged**